protocols/autonat/tests/test_server.rs

```diff
@@ -195,11 +195,14 @@ async fn test_dial_back() {
                         },
                     num_established,
                     concurrent_dial_errors,
+                    time_taken,
                 } => {
                     assert_eq!(peer_id, client_id);
                     assert_eq!(num_established, NonZeroU32::new(2).unwrap());
                     assert!(concurrent_dial_errors.unwrap().is_empty());
                     assert_eq!(address, expect_addr);
+                    assert!(time_taken>Duration::from_micros(1));
+                    assert!(time_taken<Duration::from_secs(9));
```

Comment:
I don't think we should extend existing tests but instead have a dedicated test that tests this.

That said, I am not sure how to go about that.

Reply:
This was kind of an afterthought. I added time_taken to the pattern to avoid a compiler error. Then I had a warning from it being unused. I could have written it to be ignored, but this was just as easy.

I can certainly change it to be ignored to maintain existing functionality as-is.